building/qubes-iso-building.md

```diff
@@ -1,9 +1,10 @@
 ---
 layout: doc
 title: Qubes ISO Building
-permalink: /doc/qubes-r3-building/
+permalink: /doc/qubes-iso-building/
```

Comment:
When changing permalink, please add redirect from the old one.

Reply:
Added. I think that last CI run failed due to unrelated links to some `/doc/dom0-tools/`.